kondo-lib/src/lib.rs

```diff
@@ -228,13 +230,35 @@ impl Project {
     }
 }
 
+const MINUTE: u64 = 60;
+const HOUR: u64 = MINUTE * 60;
+const DAY: u64 = HOUR * 24;
+const WEEK: u64 = DAY * 7;
+const MONTH: u64 = WEEK * 4;
+const YEAR: u64 = MONTH * 12;
+
+
+pub fn parse_old(s: &str) -> Result<u64, ParseIntError> {
+
+    fn calculate(s: &str, mult: u64) -> Result<u64, ParseIntError> {
+        Ok(s[ .. s.len() - 1].parse::<u64>()? * mult)
+    }
+
+    let c: char = s.chars().last().unwrap();
```

Comment:
:) In this app, it _shouldn't_ happen because the command line argument parser will not call it the function if there is no characters, but I'm erring on the side of conservativeness!